projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -219,24 +220,46 @@ export class IgxInputGroupComponent extends DisplayDensityBase implements IgxInp
     }
 
     /**
-     * Returns whether the input element of the input group will be automatically focused on click.
+     * @deprecated Returns whether the input element of the input group will be automatically focused on click.
      * ```typescript
      * let supressInputAutofocus = this.inputGroup.supressInputAutofocus;
      * ```
      */
+    @DeprecateProperty(`'supressInputAutofocus' @Input property is deprecated. Use 'suppressInputAutofocus' instead.`)
     @Input()
     public get supressInputAutofocus(): boolean {
-        return this._supressInputAutofocus;
+        return this.suppressInputAutofocus;
     }
 
     /**
-     * Sets whether the input element of the input group will be automatically focused on click.
+     * @deprecated Sets whether the input element of the input group will be automatically focused on click.
      * ```html
      * <igx-input-group [supressInputAutofocus]="true"></igx-input-group>
      * ```
      */
     public set supressInputAutofocus(value: boolean) {
-        this._supressInputAutofocus = value;
+        this.suppressInputAutofocus = value;
+    }
+
+    /**
+     * Returns whether the input element of the input group will be automatically focused on click.
```

Comment:
Perhaps now is the right time to clarify the description too?
```suggestion
     * Don't automatically focus the input element on click on one of the other elements in the input group such as a prefix or suffix.
```
You can also add a remark that this is particularly useful to avoid focus from bringing up the software keyboard on mobile devices when opening a dropdown or something similar.